### DISCUSSION TITLE
Disable Copilot for All Buffers

### DISCUSSION THREAD

**Discussion by LandonSchropp** [zbirenbaum/copilot.lua#219] [Q&A] [answered]
Occasionally, I like to disable Copilot completely in Neovim when I'm doing practice problems.  I know that I can run `:Copilot toggle` to disable Copilot for the current buffer, but it's easy to forget to do this when I open new problems and accidentally disable Copilot altogether.

Is there any Copilot command I can run to disable Copilot indefinitely for a Neovim process?

**Comment by MunifTanjim** [comment, marked as answer]
`:Copilot disable`
